recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
Thanks for reviewing! 
This is the only way you can tell pysdl2 where to find the sdl2.(dll|dylib|so) library it requires, and the only way you can configure this module to work in combination with virtual environments. 
Additionally, by configuring like this, we prevent "leakage", as in that the Anaconda pysdl2 module unintentionally finds a system sdl2 module, which it is not supposed to use.
In any case, changing an environment variable like this only alters it inside Python and for as long as the python process runs. The variable does persist outside of Python, so there is no real danger in doing it like this as far as I know. See [this stackoverflow post](http://stackoverflow.com/questions/716011/why-cant-environmental-variables-set-in-python-persist) for more info.